apps/admin/pulumi/index.ts

```diff
@@ -10,11 +9,44 @@ export = async () => {
         WbyEnvironment: process.env.WEBINY_ENV as string
     });
 
-    const app = new App();
-    const cloudfront = new Cloudfront({ appS3Bucket: app.bucket });
+    const bucket = new CloudFrontBucket("admin-app");
```

Comment:
I think this makes sense. But you know what, the other day @Pavel910 and I discussed the possible end goal with all of this Pulumi code, and the conclusion was: let's have a couple of these standard components that we use in Webiny projects. This way users are less mentally overloaded by the whole setup.

For example, for Admin Area, let's have the `SPA` component. This one we'd use with the Admin Area project app.

For the Website app, we wouldn't be able to use `SPA` because it has a bit different architecture. So, we could maybe create the `PageBuilderWebsite` component, that would internally utilize the `SPA` one and possibly some other components.

All in all, I think it might be better for us to immediately utilize this approach instead of doing another project upgrade down the road. Two birds with one stone.